addon/controllers/detail-edit-form.js

```diff
@@ -129,28 +129,39 @@ export default EditFormController.extend({
       @param {Boolean} skipTransition If `true`, then transition during close form process will be skipped.
     */
     close(skipTransition) {
-      if (!this.get('_hasParentRoute')) {
-        this._super.apply(this, arguments);
-        return;
-      }
+      this.close(skipTransition);
+    },
+  },
 
-      if (this.get('saveBeforeRouteLeave')) {
-        this.transitionToParentRoute(skipTransition, true);
-        return;
-      }
+  /**
+    Сlose edit form and transition to parent route.
 
-      // If 'saveBeforeRouteLeave' == false & 'close' button has been pressed,
-      // before transition to parent route we should validate model and then upload files
-      // (if some file components are present on current detail rout),
-      // because after transition, all components correspondent to current detail route (including file components)
-      // gonna be destroyed, and files won't be uploaded at all.
-      let model = this.get('model');
-      model.validate().then(() => model.beforeSave({ softSave: true })).then(() => {
-        this.transitionToParentRoute(skipTransition, false);
-      }, (reason) => {
-        this.rejectError(reason);
-      });
+    @method close
+    @param {Boolean} skipTransition If `true`, then transition during close form process will be skipped.
+  */
+  close(skipTransition) {
```

Comment:
Please add `rollBackModel` as second parameter (as now it declared in super function). It can be passed to super function when this code will be performed:
`this._super.apply(this, arguments)`
Also please add `rollBackModel` as second parameter to `close` action in `detail-edit-form` and `edit-form` controllers (it should be also passed to the `close` method of controller when it calls inside action).